pages/linux/pihole.md

```diff
@@ -1,15 +1,15 @@
 # pihole
 
 > Terminal interface for the Pi-hole ad-blocking DNS server.
-> More information: <https://pi-hole.net>.
+> More information: <https://docs.pi-hole.net/core/pihole-command/>.
 
 - Check the Pi-hole daemon's status:
 
 `pihole status`
 
-- Update Pi-hole:
+- Update Pi-hole (software and adlists):
```

Comment:
```suggestion
- Update Pi-hole and Gravity:
```

Reply:
Those are the words I was looking for, thanks for this suggestion!